streaming/src/test/java/org/apache/spark/streaming/JavaAPISuite.java

```diff
@@ -987,12 +988,12 @@ public void testPairMap2() { // Maps pair -> single
     JavaDStream<Tuple2<String, Integer>> stream = JavaTestUtils.attachTestInputStream(ssc, inputData, 1);
     JavaPairDStream<String, Integer> pairStream = JavaPairDStream.fromJavaDStream(stream);
     JavaDStream<Integer> reversed = pairStream.map(
```

Comment:
Some of these are unrelated to the PR, but just cleans up the formatting of the JavaAPISuite which is quite badly formatted.